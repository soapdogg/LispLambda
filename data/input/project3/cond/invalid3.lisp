(COND ((INT T) 34) ((NULL T) 12) ((NIL 1) 4))