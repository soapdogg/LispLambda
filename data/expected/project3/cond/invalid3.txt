Error! None of the conditions in the COND function evaluated to true.